MHCollapsibleViewManager.h

```diff
@@ -0,0 +1,65 @@
+//
+//  MHCollapsibleViewManager.h
+//  Represents a view piece that is collapsible
+//  It manages the following:
+//      The header record style/accessory for clicked record
+//      Showing/Hiding rows under header
+//      Clicked child cell style/accessory
+//      Keeps track if cell was clicked using MHFilterLabel class
+//
+//  Created by Lizzy Randall on 5/6/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface MHCollapsibleViewManager : NSObject
+
+@property (strong, nonatomic) NSMutableArray *data;
```

Comment:
objc is very verbose so when it comes to naming properties like this, the more descriptive it can be the better. I think in this case a better name might be filters